packages/angular/projects/clarity-adoption/src/index.html

```diff
@@ -10,4 +10,21 @@
   <body>
     <app-root></app-root>
   </body>
+
+  <!-- Hotjar Tracking Code for clarity.design/adoption-guide -->
+  <script>
```

Comment:
I would rather not have the feedback form on anything except the website. We don't use it on storybook either. 
If we need analytics GA is a better option.